packages/platform/atoms/event-types/wrappers/EventAvailabilityTabWebWrapper.tsx

```diff
@@ -0,0 +1,52 @@
+import { useFormContext } from "react-hook-form";
+
+import useLockedFieldsManager from "@calcom/features/ee/managed-event-types/hooks/useLockedFieldsManager";
+import { EventAvailabilityTab } from "@calcom/features/eventtypes/components/tabs/availability/EventAvailabilityTab";
+import type { EventTypeSetup, FormValues } from "@calcom/features/eventtypes/lib/types";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { trpc } from "@calcom/trpc/react";
+import useMeQuery from "@calcom/trpc/react/hooks/useMeQuery";
+
+export type EventAvailabilityTabWebWrapperProps = {
+  eventType: EventTypeSetup;
+  isTeamEvent: boolean;
+};
+
+const EventAvailabilityTabWebWrapper = (props: EventAvailabilityTabWebWrapperProps) => {
+  const { t } = useLocale();
+  const formMethods = useFormContext<FormValues>();
+  const scheduleId = formMethods.watch("schedule");
+  const { data: loggedInUser } = useMeQuery();
```

Comment:
should we move this further up the tree, we are calling this in other tabs too

Reply:
yes if it used in multiple tabs feel free to bring it in the main event type wrapper

Reply:
https://github.com/calcom/cal.com/commit/48a7df8ab740c0a59be3b7a9499565cbb70569d1